schemas/common/definitions.json

```diff
@@ -0,0 +1,257 @@
+{
+  "id": "https://schema.management.azure.com/schemas/common/definitions.json#",
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "definitions",
+  "description": "Common definitions",
+  "definitions": {
```

Comment:
>definitions [](start = 3, length = 11)

we have something in deploymentTemplateParameters.json. Do we need to copy that over?

Reply:
No - that was a deliberate omission. All of the definitions I could find in deploymentTemplateParameters.json were specific to the api-version.